pkgs/applications/video/kodi/unwrapped.nix

```diff
@@ -75,12 +75,12 @@ let
   };
 
   apache_commons_lang = fetchzip {
-    url = "https://dlcdn.apache.org//commons/lang/binaries/commons-lang3-3.14.0-bin.zip";
+    url = "https://archive.apache.org/dist/commons/lang/binaries/commons-lang3-3.14.0-bin.zip";
     sha512 = "sha512-eKF1IQ6PDtifb4pMHWQ2SYHIh0HbMi3qpc92lfbOo3uSsFJVR3n7JD0AdzrG17tLJQA4z5PGDhwyYw0rLeLsXw==";
   };
 
   apache_commons_text = fetchzip {
-    url = "https://dlcdn.apache.org//commons/text/binaries/commons-text-1.11.0-bin.zip";
+    url = "https://archive.apache.org/dist/commons/text/binaries/commons-text-1.11.0-bin.zip";
```

Comment:
```suggestion
    url = "mirror://apache/commons/text/binaries/commons-text-1.11.0-bin.zip";
```

Reply:
Thanks! Didn't know such a thing existed. Is it documented somewhere?

What bugs me a little bit is that `fetchzip` succeeds only on the 5th attempt now. But that's the price to pay for "stability", I guess.